graylog2-web-interface/src/views/components/searchbar/queryvalidation/QueryValidation.tsx

```diff
@@ -212,7 +212,7 @@ const QueryValidation = () => {
                                 $clickable
                                 tabIndex={0}
                                 type="button">
-              <ErrorIcon $status={status} name={isInfo ? 'database' : 'error'} />
+              {isInfo && <ErrorIcon $status={status} name={isInfo ? 'database' : 'error'} />}
```

Comment:
`isInfo` is always true here, so we can just set it to 'database' right?

Reply:
Good findings, this was the result of using different icon components during development iteration.